### PR TITLE
ncm-metaconfig: httpd: add davrods (webdav for irods) config

### PR DESCRIPTION
**Pull request by kwaegema**

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/httpd/2.4/tests/profiles/davrods.pan [hunk 1]
```diff
@@ -0,0 +1,58 @@
+object template davrods;
+
+include 'metaconfig/httpd/schema';
+
```
Comment (ned21) [inline ncm-metaconfig/src/main/metaconfig/httpd/2.4/tests/profiles/davrods.pan:4]: I think this needs a:
include 'components/metaconfig/config';

here to pass the travis test?

File: ncm-metaconfig/src/main/metaconfig/httpd/2.4/tests/profiles/davrods.pan [hunk 2]
```diff
@@ -0,0 +1,58 @@
+object template davrods;
+
```
Comment (ned21) [inline ncm-metaconfig/src/main/metaconfig/httpd/2.4/tests/profiles/davrods.pan:2]:
```suggestion

include 'components/metaconfig/config';
```

**Review by ned21 — COMMENTED** [review]
Can you squash 882f2772 into the previous one please? The latter is a trivial fix up so we don't need two commits in the history. Otherwise code LGTM

**Comment by kwaegema** [comment]
@ned21 squashed commit

**Comment by ned21** [comment]
> @ned21 squashed commit

Did it work right? I still see the commit I think, and also an extra merge commit.

**Comment by kwaegema** [comment]
@ned21 did something weird there, should be ok now

**Comment by ned21** [comment]
Thanks! Looks like 1a3c78f46b695bb2516b5eb4bd720a6d598dba67 is now just a single blank line being removed. Can you combine it with 44d224e0ddeec202508c765633757e3592679c74 to keep the history tidy please?

**Comment by kwaegema** [comment]
@ned21 done

**Review by ned21 — APPROVED** [review]

**Pull request merged**